<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ihm_Balise</class>
 <widget class="QMainWindow" name="ihm_Balise">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>DÃ©tection par Balise</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>PosX</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>PosY</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Teta</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_Robot1">
         <property name="text">
          <string>Robot1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="Robot1_PosX">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-9999.989999999999782</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QDoubleSpinBox" name="Robot1_PosY">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-9999.989999999999782</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QDoubleSpinBox" name="Robot1_Teta">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-9999.989999999999782</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_Robot2">
         <property name="text">
          <string>Robot2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="Robot2_PosX">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-9999.989999999999782</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QDoubleSpinBox" name="Robot2_PosY">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-9999.989999999999782</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QDoubleSpinBox" name="Robot2_Teta">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-9999.989999999999782</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_Robot3">
         <property name="text">
          <string>Robot3</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="Robot3_PosX">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-9999.989999999999782</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QDoubleSpinBox" name="Robot3_PosY">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-9999.989999999999782</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QDoubleSpinBox" name="Robot3_Teta">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-9999.989999999999782</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_Robot4">
         <property name="text">
          <string>Robot4</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="Robot4_PosX">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-9999.989999999999782</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QDoubleSpinBox" name="Robot4_PosY">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-9999.989999999999782</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QDoubleSpinBox" name="Robot4_Teta">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-9999.989999999999782</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Retours Robots</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>PosX</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>PosY</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Teta</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_Robot1_2">
         <property name="text">
          <string>Grosbot</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="Grosbot_PosX">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-9999.989999999999782</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QDoubleSpinBox" name="Grosbot_PosY">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-9999.989999999999782</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QDoubleSpinBox" name="Grosbot_Teta">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-9999.989999999999782</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_Robot2_2">
         <property name="text">
          <string>Minibot</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="Minibot_PosX">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-9999.989999999999782</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QDoubleSpinBox" name="Minibot_PosY">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-9999.989999999999782</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QDoubleSpinBox" name="Minibot_Teta">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="minimum">
          <double>-9999.989999999999782</double>
         </property>
         <property name="maximum">
          <double>9999.989999999999782</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>323</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
